storybook/stories/RowView/index.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components/native';
+
+const ViewContainer = styled.View`
+    display: flex;
+    align-items: center;
+    flex-direction: row;
+    padding-left: 50;
+    padding-right: 50;
+    justify-content: space-around;
+`;
+
+export default function RowView({ children, spaceTop = 0 }) {
+    return (
+        <ViewContainer style={{ paddingTop: spaceTop }}>
```

Comment:
Let's better pass a style prop and then when can pass any style we want in the stories:
```
<ViewContainer style={style}>
```